examples/sushi/config.py

```diff
@@ -9,12 +9,26 @@
     GatewayConfig,
     SparkConnectionConfig,
 )
+from sqlmesh.core.notification_target import (
+    BasicSMTPNotificationTarget,
+    NotificationStatus,
+    SlackApiNotificationTarget,
+    SlackWebhookNotificationTarget,
+)
 from sqlmesh.core.user import User, UserRole
 
 CURRENT_FILE_PATH = os.path.abspath(__file__)
 DATA_DIR = os.path.join(os.path.dirname(__file__), "data")
 
 
+class CustomSMTPNotificationTarget(BasicSMTPNotificationTarget):
+    def notify_run_failure(self, exc: Exception) -> None:
+        with open("/home/sqlmesh/sqlmesh.log", "r", encoding="utf-8") as f:
```

Comment:
why is this hardcoded?

Reply:
Oops, I'll remove it, although it'd be nice to showcase how to customize notifications to handle more advance use cases.